app/src/main/res/layout/view_setting_radio_button.xml

```diff
@@ -33,12 +33,28 @@
         android:importantForAccessibility="no"
         android:visibility="gone" />
 
-    <TextView
-        android:id="@+id/text"
+    <LinearLayout
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_weight="1"
-        tools:text="Simple messages" />
+        android:orientation="vertical">
+
+        <TextView
+            android:id="@+id/text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            tools:text="Simple messages" />
+
+        <TextView
+            android:id="@+id/subText"
+            style="@style/BodySmall.Tertiary"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:visibility="gone"
+            tools:text="@string/settingsDefault"
+            tools:visibility="visible"
+            android:paddingTop="@dimen/marginStandardVerySmall"/>
```

Comment:
Same comment as for the class, you don't need to put this text inside the SettingRadioButtonView, you can add a separate simple TextView outside the item